src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettings.tag

```diff
@@ -6,242 +6,231 @@
-                    </div>
-                </div>
-                <div class="row radio"><%
-                    %>
-                    <h5 class="col-xs-2"
-                        title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTSVISIBLECUSTOM %>"
-                        data-toggle="tooltip"
-                        data-placement="top">
-                        <label for="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>_custom">
-                            At
-                        </label>
-                        <input type="radio"
-                            name="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>"
-                            id="<%= Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON %>_custom"
-                            value="<%= Const.INSTRUCTOR_FEEDBACK_RESULTS_VISIBLE_TIME_CUSTOM %>"
-                            <c:if test="${additionalSettings.responseVisibleDateChecked}">checked</c:if>
-                        >
-                    </h5>
-                    <div class="col-xs-5">
-                        <input class="form-control"
-                            type="text"
-                            name="<%= Const.ParamsNames.FEEDBACK_SESSION_PUBLISHDATE %>"
-                            id="<%= Const.ParamsNames.FEEDBACK_SESSION_PUBLISHDATE %>"
-                            value="${additionalSettings.responseVisibleDateValue}"
-                            <c:if test="${additionalSettings.responseVisibleDateDisabled}">disabled</c:if>
-                        >
-                    </div>
-                    <div class="col-xs-5">
-                        <select class="form-control"
-                            name="<%= Const.ParamsNames.FEEDBACK_SESSION_PUBLISHTIME %>"
-                            id="<%= Const.ParamsNames.FEEDBACK_SESSION_PUBLISHTIME %>"
-                            title="<%= Const.Tooltips.FEEDBACK_SESSION_PUBLISHDATE %>"
-                            data-toggle="tooltip"
-                            data-placement="top"
-                            <c:if test="${additionalSettings.responseVisibleDateDisabled}">disabled</c:if>
-                        >
-                            <c:forEach items="${additionalSettings.responseVisibleTimeOptions}" var="option">
-                                <option ${option.attributesToString}>
-                                    ${option.content}
-                                </option>
-                            </c:forEach>
+      <div class="col-xs-12 col-md-6 border-left-gray" id="responsesVisibleFromColumn">
+        <div class="row">
+          <div class="col-xs-12"
+              title="<%= Const.Tooltips.FEEDBACK_SESSION_RESULTSVISIBLELABEL %>"
+              data-toggle="tooltip"
+              data-placement="top">
+            <label class="label-control">Responses visible from</label>
+          </div>
+        </div>
+        <div class="row radio"><%
+          %>
```

Comment:
It should be safe to remove the `<% %>` here, although this isn't part of the issue.